README.md

```diff
@@ -1,24 +1,19 @@
-# Tictactoe
+# Elixir Tic Tac Toe
 
-**TODO: Add description**
+[![Build Status](https://travis-ci.org/emashliles/elixir-tic-tac-toe.svg?branch=master)](https://travis-ci.org/emashliles/elixir-tic-tac-toe)
 
-## Installation
+## Running the game
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed as:
-
-  1. Add `tictactoe` to your list of dependencies in `mix.exs`:
+   You can run the game using the mix task included. 
 
     ```elixir
```

Comment:
If you look at the readme on the branch you will see that this is not showing correctly. The ```'s needs to be on new lines.